app/subscriber/src/features/search-page/components/advanced-search/components/sections/SeriesSection.tsx

```diff
@@ -20,18 +31,60 @@ export const SeriesSection: React.FC = () => {
   );
   return (
     <Row justifyContent="center">
-      <Select
-        width={FieldSize.Stretch}
-        key={filter.seriesIds?.join(',')}
-        isMulti
-        options={seriesOptions}
-        name="series"
-        onChange={(newValue: any) => {
-          const seriesIds = newValue.map((v: OptionItem<number>) => v.value);
-          storeFilter({ ...filter, seriesIds: seriesIds });
-        }}
-        value={seriesOptions.filter((o) => o.value && filter.seriesIds?.includes(+o.value))}
-      />
+      <Show visible={!displayFiltersAsDropdown}>
+        <div className="check-box-list">
+          <div className="chk-box-container chk-series select-all">
+            <Button
+              variant={ButtonVariant.link}
+              onClick={() =>
+                storeFilter({ ...filter, seriesIds: seriesOptions.map((m) => +m.value!) })
+              }
+            >
+              Select All
+            </Button>
+            /
+            <Button
+              variant={ButtonVariant.link}
+              onClick={() => storeFilter({ ...filter, seriesIds: [] })}
+            >
+              Deselect All
+            </Button>
+          </div>
+          {seriesOptions.map((item, index) => (
```

Comment:
Just a note regarding using `index` for they `key` in many of your implementations.  This is generally not a reliable key and should only be used when it's the only option.  However, there are safe use-cases for it.  I don't know whether these are safe or not.